src/Ocelot/WebSockets/Middleware/WebSocketsProxyMiddleware.cs

```diff
@@ -94,6 +94,13 @@ private static async Task Proxy(HttpContext context, string serverEndpoint)
             }
 
             var client = new ClientWebSocket();
+
+
+            if (context.Items.DownstreamRoute()?.DangerousAcceptAnyServerCertificateValidator == true)
```

Comment:
How do you identify the route from this `Items` which have enabled this cert-option?
Is `Items` collection or model class?